<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorReportView</class>
 <widget class="QWidget" name="DoctorReportView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>医生报告管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #f9f9f9;
        font-family: &quot;Arial&quot;, sans-serif;
        color: #333;
    }
    QLineEdit {
        border: 1px solid #ccc;
        border-radius: 5px;
        padding: 5px;
    }
	QComboBox{
		background-color: #fafafa;
        border: 1px solid #ccc;
        border-radius: 5px;
        padding: 8px 15px;
	}
    QPushButton {
        background-color: #fafafa;
        border: 1px solid #ccc;
        border-radius: 5px;
        padding: 8px 15px;
    }
    QPushButton:hover {
        background-color: #f3f3f3;
    }
    QTableView {
        border: 1px solid #ddd;
        background-color: white;
        gridline-color: #eee;
        selection-background-color: #cce5ff;
        selection-color: #000;
    }
    QLabel {
        font-size: 14px;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="textSearch">
       <property name="placeholderText">
        <string>请输入姓名进行查找</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btSearch">
       <property name="text">
        <string>查找</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/search.png</normaloff>:/img/search.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btAdd">
       <property name="text">
        <string>添加</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/add.png</normaloff>:/img/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btDelete">
       <property name="text">
        <string>删除</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/delete.png</normaloff>:/img/delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btEdit">
       <property name="text">
        <string>修改</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/edit.png</normaloff>:/img/edit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="createdTimeBox">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <item>
          <property name="text">
           <string>按创建时间顺序</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按创建时间逆序</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="btimport">
           <property name="text">
            <string>导入</string>
           </property>
           <property name="icon">
            <iconset resource="lab4.qrc">
             <normaloff>:/img/import.png</normaloff>:/img/import.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="styleSheet">
        <string notr="true"> QTableView {
        border: 1px solid #dcdcdc;
        border-radius: 5px;
        background-color: #fafafa;
       }
       QTableView::item {
        padding: 8px;
       }
       QTableView::horizontalHeader {
        background-color: #2196F3;
        color: white;
       }
       QTableView::verticalHeader {
        background-color: #f0f0f0;
       }
       QTableView::item:selected {
        background-color: #4CAF50;
        color: white;
       }</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btPrev">
       <property name="text">
        <string>上一页</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/prev.png</normaloff>:/img/prev.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalAndCurrent">
       <property name="text">
        <string>第Y页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btNext">
       <property name="text">
        <string>下一页</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/next.png</normaloff>:/img/next.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lab4.qrc"/>
 </resources>
 <connections/>
</ui>
